docs/UsersGuide/source/ContributorsGuide.rst

```diff
@@ -0,0 +1,369 @@
+    +------------------+------------------------------------------------+
+    | NCAR             | Mike Kavulich (@mkavulich)                     |
+    |                  |                                                |
+    |                  | Will Mayfield (@willmayfield)                  |
+    |                  |                                                |
+    |                  | Jamie Wolff (@jwolff-ncar)                     |
+    +------------------+------------------------------------------------+
+    | NSSL             | Yunheng Wang (@ywangwof)                       |
+    +------------------+------------------------------------------------+
+
+
+.. _ContribProcess:
+
+Contribution Process
+========================
+
+The following steps should be followed in order to make changes to the ``develop`` branch of the ``ufs-srweather-app`` repository. Communication with code managers and the code management team throughout the process is encouraged.
+
+    #. **Issue** - Open an issue to document changes. Click `here <https://github.com/ufs-community/ufs-srweather-app/issues/new/choose>`__ to open a new ``ufs-srweather-app`` issue or see :numref:`Step %s <Issue>` for detailed instructions. 
+    #. **GitFlow** - Follow `GitFlow <https://nvie.com/posts/a-successful-git-branching-model/>`__ procedures for development. 
+    #. **Fork the repository** - Read more `here <https://docs.github.com/en/get-started/quickstart/fork-a-repo>`__ about forking in GitHub.
+    #. **Create a branch** - Create a branch in your fork of the authoritative repository. Follow `GitFlow <https://nvie.com/posts/a-successful-git-branching-model/>`__ conventions when creating the branch. Branches should be named as follows, where [name] is a one-word description of the branch:
+
+        * **bugfix/[name]:** Fixes a demonstrably incorrect portion of code
+        * **feature/[name]:** Adds a new feature to the code
+        * **enhancement/[name]:** Improves an existing portion of the code
+        * **textonly/[name]:** Changes elements of the repository that do not impact program output or log files (e.g., changes to README, documentation, comments, changing quoted Registry elements, white space alignment). Any change which does not impact the compiled code in any way should fall under this category.
+        ..
+            COMMENT: Ask for input on *textonly* description. 
+    #. **Development** - Perform and test changes in the branch. Document work in the issue and mention the issue number in commit messages to link your work to the issue (e.g. commit -m "Issue #23 - ...commit message..."). Test code modifications on as many platforms as possible, and request help with further testing from code review committee when unable to test on all platforms. Document changes to the workflow and capabilities (either in the ``.rst`` files or separately) so that SRW App documentation stays up-to-date. 
```

Comment:
No-the #. is a numbering convention so that the list numbers are automatically populated. The comment above it will mess with formatting/spacing a bit until removed, but it's showing up just fine on RTD.

Reply:
OK, yeah, I was seeing a different color of text below the comment, so I wanted to make sure.  Thanks.